bounce/db/__init__.py

```diff
@@ -19,6 +19,12 @@ def create_engine(driver, user, password, host, port, db_name):
         f'{driver}://{user}:{password}@{host}:{port}/{db_name}', echo=True)
 
 
+def configure_mappers():
+    """Link club, user, and membership classes with table metadata
+    before creating tables. Needed for search functionality"""
+    return sqlalchemy.orm.configure_mappers()
```

Comment:
Based on the [documentation](https://sqlalchemy-searchable.readthedocs.io/en/latest/installation.html#quickstart) it looks it's assumed that we'll be defining our schema with SQLAlchemy, which is not the case. That being said, I don't think calling `configure_mappers` is going to do anything for us.

Reply:
alright I'll get rid of that part